### PR TITLE
i18n(es): Update `images`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Updated `images` based on this commit 3fede270aeaf220771e86da0e6f6ee2ad12f045c

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/images.mdx [hunk 1]
```diff
@@ -649,6 +649,16 @@ Si la imagen es meramente decorativa (es decir, no contribuye a la comprensión
...
 
+:::note
+Cuando se utiliza un [administrador de paquetes estricto](https://pnpm.io/pnpm-vs-npm#npms-flat-tree) como `pnpm`, es posible que debas instalar manualmente Sharp en tu proyecto, aunque sea una dependencia de Astro:
+
+```bash
+pnpm install sharp
+```
+:::
+
+### Configure Squoosh
```
Comment (Waxer59) [inline src/content/docs/es/guides/images.mdx:660]:
```suggestion
### Configura Squoosh
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR, te dejo una sugerencia 🤠!

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**